autopush/logging.py

```diff
@@ -69,14 +70,12 @@ def start(self):
 def stdout(message):
     """Format a message appropriately for structured logging capture of stdout
     and then write it to stdout"""
-    # uncomment to get concise human readable logging messages.
-    """
-    if message['error']:
-        sys.stdout.write("ERROR: %s\n" % message['message'])
-    else:
-        sys.stdout.write("       %s\n" % message['message'])
-    return
-    # """
+    if HUMAN:
+        if message['error']:
```

Comment:
It's pretty much guaranteed. Plus we're doing the same thing later in the code.
https://github.com/mozilla-services/autopush/blob/master/autopush/logging.py#L84